aetest/instance_vm.go

```diff
@@ -206,9 +210,13 @@ func (i *instance) startChild() (err error) {
 	}
 	appserverArgs = append(appserverArgs, filepath.Join(i.appDir, "app"))
 
-	i.child = exec.Command(python,
-		appserverArgs...,
-	)
+	if len(os.Getenv("APPENGINE_DEV_APPSERVER_BINARY")) > 0 {
```

Comment:
If the user is specifying the binary to use, then we shouldn't run the checks for if python and dev_appserver.py exist up on lines 159-170.